### ISSUE TITLE
Feature request: - -o json should just be json

### ISSUE THREAD

**Issue by LittleColin** [mivano/azure-cost-cli#88]
It would be helpful if the `-o json` caused just json to be sent to stdout but instead it also includes fetch messages. Can the fetch messages be sent to stderr instead?

```
Fetching subscription details...
Fetching cost data...
Fetching forecasted cost data...
Fetching cost data by service name...
Fetching cost data by location...
Fetching cost data by resource group...
{
   "totals"...
```

**Comment by mivano** [comment]
Ai, that is indeed not correct. I will have to look into this. Thanks for reporting!

**Comment by mivano** [comment]
I just tried it locally and did not see this effect. 
<img width="395" alt="image" src="https://github.com/mivano/azure-cost-cli/assets/668535/3c0c2850-0717-4f5b-9588-d661dab79cf8">

Can you give me the exact command you are running?

**Comment by akashkroy** [comment]
I am facing this issue as well . Command Used : 
```
azure-cost dailyCosts -o json > cost.json
```
When I cat it, the output is fine because the terminal is able to process these control characters.  Try opening it in vim or another text editor, you will see these weird control characters. 
It becomes an issue when you try to read this output file programmatically
![image](https://github.com/mivano/azure-cost-cli/assets/25970310/7ec8e37f-710e-4816-907d-a80b41f12087)

**Comment by mivano** [comment]
Yeah, that is not correct. Will need to look into it. Thanks for reporting!

**Comment by codepic** [comment]
@mivano I had a look at the code and probably you need to skip the [AnsiConsole.Status](https://github.com/mivano/azure-cost-cli/blob/cbf5a1106739c8e581c4226515f7db539dc724e4/src/Commands/DailyCost/DailyCost.cs#L89) when output is set to `JSON`.

**Comment by mivano** [comment]
Thanks for the pointers @codepic. I was able to reproduce. I do want to keep using the status overview, so I added a custom implementation that checks for a redirected output. In that case, the status is not written to the console.

This is released in 0.36. 

Thanks again for raising the issue and I hope this fixes the problem.
